docker-hub/builds/index.md

```diff
@@ -31,6 +31,11 @@ pushing to the registry. You can use these tests to create a continuous
 integration workflow where a build that fails its tests does not push the built
 image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
 
+Depending on your plan, you may get parallel builds, which means that `N`
+autobuilds can be run at the same time. `N` is configured by the plan that you
+subscribe to. Once `N+1` builds are running, any additional builds are
+queued to be run later.
```

Comment:
```suggestion
queued to be run later.  The total number of pending builds in the queue is limited to 30 and further requests will be
ignored
```

Reply:
Thanks Arko. I'll do a follow-up PR for this update.

Reply:
thanks maybe we can use `discarded` instead of `ignored`

Reply:
@arkodg https://github.com/docker/docker.github.io/pull/11689 - PTAL